src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx

```diff
@@ -140,10 +140,17 @@ export default function RapidResponsePersonnelTable(props: Props) {
                 'country_from',
                 strings.personnelTableDeployedParty,
                 (item) => item.country_from?.society_name,
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (isDefined(item.country_from?.record_type === 3)) {
+                        return {
+                            to: undefined,
+                        };
```

Comment:
Let's add a note

Reply:
Let's use a constant to denote region type.
For example 
export const TYPE_IMMINENT = 0 satisfies TypeOfDrefEnum;